packages/compass-crud/src/stores/crud-store.js

```diff
@@ -1116,3 +1118,6 @@ async function fetchDocuments(dataService, ns, filter, findOptions) {
   return find(ns, filter, findOptions);
 }
 
+function resultId() {
+  return Math.floor(Math.random() * Math.pow(2, 64));
```

Comment:
```suggestion
  return Math.floor(Math.random() * (2 ** 53));
```

since otherwise the result won’t be uniform anymore – but why not just use a plain counter here? That practically eliminates the risk of collisions entirely, and makes the values a bit easier for humans to identify.

Reply:
I started with a counter but it is weird because this is an arbitrary crud result and the the latest crud result would have to keep track of how many results there have been in total. OR I'd have to get that from elsewhere but the react actions for making a query are already weirdly in the same place as the results for _a_ query.. OR I'd have to refactor all of compass-crud to make sense..

I'll do the 2 ** 53 thing.